### PR TITLE
Modifiable persistent deployment configuration

### PR DESCRIPTION
**Pull request by michaelfig**
This change is just to write out the `ag-setup-cosmos init` configuration to `deployment.json`.  Also allow rerunning `init` to modify the `deployment.json`.

Bonus change: upgrade the DigitalOcean default node size to 4GB RAM, which is necessary to meet system requirements.

### CODE REVIEW

**Comment by dckc** [comment]
I'm struggling to figure out which end is up... `init.js` seems to export `doInit`... which is imported from `main.js`... which exports `main` but... then the trail goes cold. I checked README.md and package.json and Makefile but I don't see how init.js gets invoked.

Also... the exported `doInit` closes over powerful globals, which makes me reluctant to give this an "I'm happy to maintain it" approval. ref #2160

**Comment by michaelfig** [comment]
> I'm struggling to figure out which end is up... `init.js` seems to export `doInit`... which is imported from `main.js`... which exports `main` but... then the trail goes cold. I checked README.md and package.json and Makefile but I don't see how init.js gets invoked.

The entrypoint is `deployment/ag-setup-cosmos` .

> Also... the exported `doInit` closes over powerful globals, which makes me reluctant to give this an "I'm happy to maintain it" approval. ref #2160

Yeah, duly noted.  This package is messy mostly because it's a glorified sysadmin script.  I'm saving a major refactoring until we make a concerted effort towards ocap discipline, but there are bigger fish to fry in the more user-facing packages.

**Review by dckc — APPROVED** [review]
I'm approving based on understand the motivation for this change.

If you a review of the implementation details is important, you should probably dismiss this review and either (a) stand by until I find time to study it further (could be a while), (b) give me a call, or (c) get another reviewer involved.

**Pull request merged**
